src/main/java/nextstep/ladder/entity/ScoreBoard.java

```diff
@@ -6,29 +6,31 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
+import nextstep.ladder.exception.UserException;
+import nextstep.ladder.exception.UserExceptionMesssage;
 import nextstep.ladder.utils.StringUtils;
 
 public class ScoreBoard {
-	private final Map<String, String> result = new LinkedHashMap<>();
+	private final Map<String, Prize> result = new LinkedHashMap<>();
 
 	public ScoreBoard(Users users, PrizeResult prizeResult) {
 		List<User> userList = users.getUsers();
 		List<Prize> prizes = prizeResult.getPrizes();
 
 		for (User user : userList) {
-			result.put(user.getName(), prizes.get(user.getPosition().getPosition()).getName());
+			result.put(user.getName(), prizes.get(user.getPosition()));
 		}
-
-		String all = result.entrySet()
-			.stream()
-			.map(entry -> StringUtils.joining(entry.getKey(), " : ", entry.getValue()))
-			.collect(Collectors.joining("\n"));
-
-		result.put("all", all);
 	}
 
-	public String getPrizeResultStr(String userName) {
+	public Prize getPrizeResultStr(String userName) {
 		return Optional.ofNullable(result.get(userName))
-			.orElseGet(() -> "참가한 사용자가아닙니다.");
+			.orElseThrow(() -> new UserException(UserExceptionMesssage.NO_PARTICIPANT));
+	}
+
+	public String getAllResult() {
+		return result.entrySet()
+			.stream()
+			.map(entry -> StringUtils.joining(entry.getKey(), " : ", entry.getValue().getName()))
+			.collect(Collectors.joining("\n"));
```

Comment:
OS 마다 줄바꿈을 인식하는게 다르기 때문에 jdk 1.7부터 도입된 현재 OS에 맞는 줄바꿈을 얻어오도록 하면 좋을 것 같아요!
```suggestion
			.collect(Collectors.joining(System.lineSeperator()));
```

Reply:
지난번 리뷰때 주셨는데 제가 깜빡했네요 ㅠ 

감사합니다